packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-wrapper.tsx

```diff
@@ -27,7 +30,6 @@ const DurationWrapper = observer(() => {
         expiry_epoch,
         expiry_time,
         start_date,
-        market_open_times,
```

Comment:
Non of the children were using `market_open_times`

Reply:
@kate-deriv could this be used in order to trigger the re-render of AdvancedDuration and Duration when this prop gets updated? Please check if the component is still re-rendered properly without it :)

Reply:
Yep, you are right, I noticed that `Expiry text` is not updated properly without receiving `market_open_times`. Should I add some comment that this prop is needed to trigger re-render?